### ISSUE TITLE
Truncate long output in docs

### ISSUE THREAD

**Issue by navdeeprana** [SciML/DiffEqDocs.jl#718]
At places, the `solve()` output are pretty long, basically a bunch of numbers that doesn't add much to the docs apart from vertical scrolling. Is there a way to truncate/abbreviate such outputs? Examples below:

- https://docs.sciml.ai/DiffEqDocs/stable/getting_started/
![image](https://github.com/SciML/DiffEqDocs.jl/assets/17153984/96e667b7-782c-4165-ac6d-65c7842d7524)
![image](https://github.com/SciML/DiffEqDocs.jl/assets/17153984/9d22b7d8-d9c5-402e-bfdb-d82ab40db1f4)
![image](https://github.com/SciML/DiffEqDocs.jl/assets/17153984/7ffe5f4f-0279-4ea7-a03b-1d16f94b33b4)

**Comment by ChrisRackauckas** [comment]
Yeah we could truncate it. I don't know of an easy way to do it though.

**Comment by navdeeprana** [comment]
How about using `summary()` for `t` and `u`? But then we only get basic information about type and size.
